### PR TITLE
Add WEIGHTS_JSON file as cli option

### PR DESCRIPTION
**Pull request by jayc13**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cypress-parallel-demo",
-  "version": "1.0.0",
+  "version": "0.10.0",
```
Comment (tnicola) [inline package.json:3]: This is just the demo package, there's no need to change the version

**Review by tnicola — CHANGES_REQUESTED** [review]
Hi Jay, thanks for this pull request, I've added a minor comment. Could you also please update the doc?

**Comment by jayc13** [comment]
> Hi Jay, thanks for this pull request, I've added a minor comment. Could you also please update the doc?

Documentation updated! Ready to go 🚀

**Review by tnicola — APPROVED** [review]

**Comment by jayc13** [comment]
Screen Shot 2022-06-13 at 6 03 38 PM
@tnicola  I can't merge it

**Comment by tnicola** [comment]
@jayc13 merged!Thanks for your contribution! :)

**Pull request merged**